packages/terra-table/src/terra-dev-site/doc/example/TableSection.jsx

```diff
@@ -0,0 +1,19 @@
+import React from 'react';
+/* eslint-disable import/no-extraneous-dependencies, import/no-unresolved, import/extensions */
+import Table, { Row, Cell, Section } from 'terra-table/lib/index';
+import Placeholder from 'terra-doc-template/lib/Placeholder';
+/* eslint-enable import/no-extraneous-dependencies, import/no-unresolved, import/extensions */
+
+const TableRowExample = () => (
+  <Table>
+    <Section title="Section Title">
```

Comment:
lets add `colSpan` since it's a required prop

Reply:
Updated.